### PR TITLE
008 pagination

### PR DESCRIPTION
**Pull request by Conaill4**

### CODE REVIEW

File: src/services/JobRoleService.ts [hunk 1]
```diff
@@ -1,27 +1,34 @@
...
-            throw new Error('Failed to get any Jobs')
-        }     
-        throw new Error("Unknown error occurred");
+    } catch (e) {
+        if (axios.isAxiosError(e)) {
+            if (e.response?.status === 404) {
+                throw new Error('No job roles open');
+            }
+            else if (e.response?.status === 500) {
+                throw new Error('Server Error');
```
Comment (mlynch147) [inline src/services/JobRoleService.ts:20]: update to say "Unknown Server Error"

File: test/unit/JobRolesServiceTests.ts [hunk 2]
```diff
@@ -71,9 +76,8 @@ describe('JobRoleService', function () {
           
           const token = "123";
```
Comment (mlynch147) [inline test/unit/JobRolesServiceTests.ts]: check if used - if not, delete

**Review by mlynch147 — CHANGES_REQUESTED** [review]
two minor comments... great stuff

File: src/services/JobRoleService.ts (hunk 1 above)
Comment (Conaill4) [reply src/services/JobRoleService.ts:20]: Updated if to handle unknown errors

File: test/unit/JobRolesServiceTests.ts (hunk 2 above)
Comment (Conaill4) [reply test/unit/JobRolesServiceTests.ts]: updated

**Review by ldevlin559 — APPROVED** [review]

**Review by mlynch147 — APPROVED** [review]

**Pull request merged**